raven/processes/wps_raven.py

```diff
@@ -69,6 +69,7 @@ def _handler(self, request, response):
 
                 # Namedtuples
                 if name in self.tuple_inputs:
+                    print(obj.data)
```

Comment:
I'd be careful about leaving this `print` function here.

Reply:
Good catch. Removed.